src/client/driver/command-executors/ensure-element-utils.js

```diff
@@ -0,0 +1,68 @@
+import hammerhead from '../deps/hammerhead';
+import testCafeCore from '../deps/testcafe-core';
+import {
+    ActionElementNonEditableError,
+    ActionElementNonContentEditableError,
+    ActionRootContainerNotFoundError,
+    ActionElementNotTextAreaError
+} from '../../../errors/test-run';
+
+
+var nativeMethods = hammerhead.nativeMethods;
+
+var waitFor         = testCafeCore.waitFor;
+var positionUtils   = testCafeCore.positionUtils;
+var domUtils        = testCafeCore.domUtils;
+var contentEditable = testCafeCore.contentEditable;
```

Comment:
Well, actually it's bad. We've used it the wrong way. Initially we've used it there we can't rich internal object, e.g.:
```js
import hammerhead from 'hammerhead';

var someMethod = hammerhead.someUtil.someMethod;
```
It's kinda makes sense there, but then this approach erroneously spread all other the code. We don't have time currently to refactor it, but at least we can avoid in the new code.